portal/portal-render-engine-impl/impl/src/java/org/sakaiproject/portal/charon/velocity/VelocityPortalRenderEngine.java

```diff
@@ -92,12 +92,14 @@ public void init() throws Exception
 			styleAble = serverConfigurationService.getBoolean("portal.styleable", false);
 			styleAbleContentSummary = serverConfigurationService.getBoolean(
 					"portal.styleable.contentSummary", false);
-			defaultSkin = serverConfigurationService.getString("portal.templates", "morpheus");
+			
+			//this is rather the template folder name then what we would refer to as skin
+			defaultSkin = serverConfigurationService.getString("portal.templates", "trinity");
+			defaultSkin = "morpheus"; 
```

Comment:
Why are you setting this as configurable and then overriding it immediately?

Why not leave the default as morpheus and then override it locally in sakai.properties, as it was originally designed?

Reply:
I did this for development purposes. This allows to switch between morpheus and trinity without restarting the tomcat (when the properties are loaded). So you can just edit that line, recompile and you switched the templates already. So that's something I would definitely remove later.
If it's a better practice that someone working on it adds that himself and I can remove it directly.